cmd/zt_scenario_helpers_for_test.go

```diff
@@ -909,3 +909,28 @@ func getDefaultRemoveRawInput(src string) rawCopyCmdArgs {
 		includeDirectoryStubs:          true,
 	}
 }
+
+func getDefaultSetPropertiesRawInput(src string, accessTier common.BlockBlobTier) rawCopyCmdArgs {
+	fromTo := common.EFromTo.BlobNone()
+	srcURL, _ := url.Parse(src)
+
+	if strings.Contains(srcURL.Host, "file") {
+		fromTo = common.EFromTo.BlobFSNone()
```

Comment:
1) Look into how to use the location detection from copy-- The URIs `https://myfileaccount.blob.core.windows.net` and `https://mydfsaccount.blob.core.windows.net` would fail this check.

2) BlobFS for Files? That smells wrong.

Reply:
`srcLocationType := InferArgumentLocation(src)` used.
Resolved for test helper function